nautobot/extras/models/jobs.py

```diff
@@ -301,6 +291,20 @@ def job_task(self):
         except TypeError as err:  # keep 2.0-2.2.2 exception behavior
             raise NotRegistered from err
 
+    @property
+    def task_queues(self):
```

Comment:
Probably need to explicitly add this to the REST serializer (and perhaps GraphQL) for backwards API compatibility.

Reply:
Added